core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -847,6 +847,13 @@ private void unparseOver(SqlWriter writer, SqlCall call, final int leftPrec,
   private void unparseItem(SqlWriter writer, SqlCall call, final int leftPrec) {
     call.operand(0).unparse(writer, leftPrec, 0);
     final SqlWriter.Frame frame = writer.startList("[", "]");
+
```

Comment:
Did you test the mig code with this jar?

Reply:
Yes